### PR TITLE
Going Global and Registration Open

### PR DESCRIPTION
**Pull request by tnabtaf**
Site wide update, publishing that we are open for registration, and that we are going global.  Also links to schedule web site.

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -150,10 +152,10 @@ partnersC4sponsorsButton: "Become a sponsor"
 subscribeImage: "subscribe.jpg"
 subscribeTitle: "Subscribe for updates"
 subscribeAction: ""
-subscribeInfo: "Registration will open in March 2020. Stay tuned!"
+subscribeInfo: "Registration is open!"
 # Tickets Block
 ticketsTitle: "Registration fees"
-ticketsInfo: "Registration grants access to all conference sections, coffee breaks, lunch and social activities. Accommodation is NOT included in the ticket price."
+ticketsInfo: "There are separate registrations for training, the confernce, and the CoFest events."
```
Comment (heuermh) [inline _config.yml]: confernce &rarr; conference

File: _includes/schedule.html [hunk 2]
```diff
@@ -1,14 +1,21 @@
...
+        </p>
+        <p style="text-align: left;"><a href='../training/'>Training</a> will be offered on July 17-18 in the Western hemishpere, and July 19-20 in the Eastern hemisphere. All training will be live and online.  Some topics will only offered in one hemisphere or the other.  Registration will be on a per session basis.  Training will use online platforms and almost all topics will be hands-on.</p>
+        <p style="text-align: left;">The main meeting will be held July 19-21 in the West, and July 20-22 in the East.  The same presentations will be offered in both hemispheres. We are seeking ways to also have posters, demos, and birds-of-a-feather span both hemispheres.</p>
+        <p style="text-align: left;">The <a href="../cofest/">CollaborationFest</a> will follow the meeting, and span the entire world (nothing new there).  There will be core communities during both windows, but we are also exploring other options for coordinating efforts continuosly.</p>
```
Comment (heuermh) [inline _includes/schedule.html]: continuosly &rarr; continuously

File: _includes/training.html [hunk 3]
```diff
@@ -3,28 +3,8 @@
...
-                                <p>{{ topic.instructors }}</p>
-                                <p><strong>Description</strong></p>
-                                <p>{{ topic.information }}</p>
-                                <p><strong>Prerequisites</strong></p>
-                                <p>{{ topic.prereqs }}</p>
-                            </div>
-                    {% endfor %}
-                </div>
-            </div>
+            <p style="text-align: left;">Training is offered over two days: July 17-18 in the <a href='https://bcc2020.sched.com/venue/West'>West</a>, and July 18-19 in the <a href='https://bcc2020.sched.com/venue/East'>East</a>. Training covers a variety of topics, and is offered thoughout both days.  Each session is 2.5 hours long, and you regiester for each session individually.
```
Comment (heuermh) [inline _includes/training.html]: regiester &rarr; register

File: _posts/2020-04-20-registration-open.markdown [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+In addition, [BCC2020 is going global, and **BCC2020 events will be held twice](going-blobal)**: once in their [original Toronto time zone](https://bcc2020.sched.com/venue/West) (BCC West), and again [12 hours later in the Eastern hemisphere](https://bcc2020.sched.com/venue/East) (BCC East).
+
+*We hope these two changes will make BCC2020 accessible to the entire world.*  Here's the plan.
+
+**Before you regsiter**
+
+1. If you are planning on attending [Training](/training/), please review the [training schedule](https://bcc2020.sched.com/) to see what topics you are interested in.  Be sure to 
+  * Set your time zone ([Toronto](https://bcc2020.sched.com/?timezone=America/Toronto), [Sydney](https://bcc2020.sched.com/?timezone=Australia/Sydney), [*lots* of others](https://bcc2020.sched.com/))
+    * And set your venue too ([West](https://bcc2020.sched.com/venue/West), [East](https://bcc2020.sched.com/venue/East)), or not &mdash; if you live somewhere in between
+2. Be aware that the East training schedule is not yet finalized.  If you are in the east, you shouldhold off until the end of April to register.
```
Comment (heuermh) [inline _posts/2020-04-20-registration-open.markdown]: shouldhold &rarr; should hold

**Review by heuermh — CHANGES_REQUESTED** [review]
Minor typos

File: _posts/2020-04-20-registration-open.markdown (hunk 4 above)
Comment (tnabtaf) [reply _posts/2020-04-20-registration-open.markdown]: Thanks.  Fixed, I think.

**Review by heuermh — APPROVED** [review]

**Comment by heuermh** [comment]
Thank you, @tnabtaf @denis-yuen @morgantaschuk!

**Pull request merged**